proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -64,11 +64,11 @@ service Query {
         "/interchain_security/ccv/provider/registered_consumer_reward_denoms";
   }
 
-  // QueryAllPairsValConAddrByConsumerChainID returns a list of pair valconsensus address
+  // QueryAllPairsValConsAddrByConsumer returns a list of pair valconsensus address
```

Comment:
nit: why not use`QueryAllPairsValConsAddrByConsumerID`?

Reply:
Because it's too long of a name for something that doesn't matter. Nobody really uses this name. The relevant parts are the REST endpoint or CLI command.